src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -31,13 +37,22 @@ type EventTrigger = backend.EventTrigger & {
   serviceAccountEmail?: string | null;
 };
 
+type ScheduleTrigger = backend.ScheduleTrigger & {
+  retryConfig?: {
+    maxRetryDuration?: string | null;
+    minBackoffDuration?: string | null;
+    maxBackoffDuration?: string | null;
+  } | null;
+};
+
 export type ManifestEndpoint = Base &
   backend.Triggered &
   Partial<backend.HttpsTriggered> &
   Partial<backend.CallableTriggered> &
   Partial<{ eventTrigger: EventTrigger }> &
   Partial<backend.TaskQueueTriggered> &
   Partial<backend.BlockingTriggered> &
+  Partial<{ scheduleTrigger: ScheduleTrigger }> &
```

Comment:
Can we add a note on why we needed to make duplicate declartion of ScheduleTrigger (*Seconds vs *Duration)?

Reply:
Added on L40